cmd/konnectd/serve.go

```diff
@@ -178,3 +190,24 @@ func serve(cmd *cobra.Command, args []string) error {
 	logger.Infoln("serve started")
 	return srv.Serve(ctx)
 }
+
+func envOrDefault(name string, def string) string {
+	v := os.Getenv(name)
+	if v == "" {
+		return def
+	}
+
+	return v
+}
+
+func listEnvArg(name string) []string {
+	list := make([]string, 0)
+	for _, keyFn := range strings.Split(os.Getenv(name), " ") {
+		keyFn = strings.TrimSpace(keyFn)
+		if keyFn != "" {
+			list = append(list, keyFn)
+		}
+	}
+
+	return list
+}
```

Comment:
Can you please move these two function to its own file (like `env.go`) or similar.